src/ape_accounts/_cli.py

```diff
@@ -8,8 +8,10 @@
 from ape.cli import ape_cli_context, existing_alias_argument, non_existing_alias_argument
 from ape_accounts import KeyfileAccount
 
-# NOTE: Must used the instantiated version of `AccountsContainer` in `accounts`
-container = accounts.containers["accounts"]
+
+def _get_container():
```

Comment:
So this was causing errors when building the docs, specifically keyerror.

Shouldn't matter - just make it access the dict more lazily by wrapping it in a method.